packages/storage/src/implementation/constants.ts

```diff
@@ -42,6 +42,11 @@ export const DEFAULT_MAX_OPERATION_RETRY_TIME = 2 * 60 * 1000;
  */
 export const DEFAULT_MAX_UPLOAD_RETRY_TIME = 10 * 60 * 1000;
 
+/**
+ * 1 second
+ */
+export const DEFAULT_MIN_SLEEP_TIME = 1000;
```

Comment:
I know we don't currently do this but it would be nice to suffix time-related constants with their units. RETRY_MILLIS is a lot more parseable than RETRY_TIME

Reply:
Ended up adding `_MILLIS` to the end of this, but I think we should wait to rename the rest in a different PR